AdvancedTeleport-Bukkit/src/main/java/io/github/niestrat99/advancedteleport/config/NewConfig.java

```diff
@@ -66,7 +66,7 @@ public class NewConfig extends ATConfig {
     public ConfigOption<Integer> MAXIMUM_TELEPORT_DISTANCE;
     public ConfigOption<Boolean> MONITOR_ALL_TELEPORTS;
     public PerCommandOption<Integer> DISTANCE_LIMITS;
-    public ConfigOption<ConfigurationSection> CUSTOM_DISTANCE_LIMITS;
+    public ConfigOption<CMConfigSection> CUSTOM_DISTANCE_LIMITS;
```

Comment:
The ConfigSection interface should be used here instead of the implementation (CMConfigSection), is there a reason this cannot be used?

Reply:
Wasn't aware of the interface type, i was just going off the exception and what it was expecting ill have that changed